tests/acceptance/blockchain/ape-config.yml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: nucypher-contracts
+    github: nucypher/nucypher-contracts
+    ref: main
```

Comment:
I didn't understand well why we are adding this ape config file here.

Reply:
I think these lines are green in this PR from a formatting change, but this is part of how nucypher-contracts is configured as a test dependency for this codebase.

Reply:
oh, true 👍